### PR TITLE
pbTests: Update Vagrantfile paths in VPC/vmDestroy

### PR DESCRIPTION
**Pull request by Willsparker**
Fixes: #2215 

VPC currently isn't running cleanly, due to a new issue with `vmDestroy.sh`. While I can't see that it's related to #2211 , `vpc.sh`, it suddenly started being an issue after this was merged. Irrespective, `vpc.sh` needs to be updated to reflect the changes in that PR anyway.

Keeping in Draft until VPC runs are (mostly) green: 
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/1183/

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [x] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: ansible/pbTestScripts/vmDestroy.sh
Comment (sxa) [inline ansible/pbTestScripts/vmDestroy.sh:98]: Ensure no syntax error when it's blank
```suggestion
	if [[ "$ID" != "" ]]; then
```

**Review by sxa — APPROVED** [review]
Approving subject to the VPC run passing

**Review by sxa — CHANGES_REQUESTED** [review]
Actually, not approving as I've got one change to add some quotes

File: ansible/pbTestScripts/vmDestroy.sh
Comment (Willsparker) [reply ansible/pbTestScripts/vmDestroy.sh:98]: Good spot :-)

**Review by karianna — APPROVED** [review]
LGTM - I should have scanned for this last time

**Comment by sxa** [comment]
@karianna Still not working yet as per the VPC link in the description :-)
@willsparker are you around today to look at it?

**Comment by sxa** [comment]
Better - the build is succeeding on the Linux platforms but tests are failing:
```
17:32:45 TESTDIR: /home/vagrant/testLocation/openjdk-tests is invalid. Please use --testdir|-t to set valid TESTDIR under aqa-tests. Default value current dir (pwd) is used if not provided.
17:32:46 testJDK.sh: line 15: cd: /home/vagrant/testLocation/openjdk-tests/TKG: No such file or directory
```
Windows playbook failed on one of the roles so that's unrelated to this I think
Solaris playbook failed with: `14:41:22 fatal: [127.0.0.1]: FAILED! => {"msg": "Timeout (62s) waiting for privilege escalation prompt: sudo: unable to resolve host adoptopenjdkSol10: node name or service name not known\r\r\n"}` (Also likely unrelated

**Comment by sxa** [comment]
Rebased (to include the fix to the test repository location) and retesting at https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/1193/

**Comment by sxa** [comment]
Looking better so I'm going to take this out of draft and approve/merge. Notes:
- https://github.com/adoptium/infrastructure/pull/2235/files should be canceled as it has a subset of the fixes in here
- Solaris runs are still failing to connect to do the build/test
- Windows runs are still failing due to https://github.com/adoptium/infrastructure/issues/2178 (Or its dup at https://github.com/adoptium/infrastructure/issues/2231)

**Comment by sxa** [comment]
Looking better so I'm going to take this out of draft and approve/merge. Notes:
- https://github.com/adoptium/infrastructure/pull/2235/files should be canceled as it has a subset of the fixes in here
- Solaris runs are still failing to connect to do the build/test
- Windows runs are still failing due to https://github.com/adoptium/infrastructure/issues/2178 (Or its dup at https://github.com/adoptium/infrastructure/issues/2231)

**Comment by sxa** [comment]
I'm going to merge this. Please @gdams @karianna can we be careful about making and merging changes going in and make sure things are tested in a selection of environments first. We can't just chuck things into the repo and especially not ignore the side-effects in the future. I'm going to get much tougher on this even if it mandates having a longer review process because we can't allow checks broken for weeks. The right course of action here would have been to merge the revert PR and wait until it could be merged without any problems.

Much as we have survived without them in the past and I'd prefer not to have to enforce extra bureacracy, if things like this occur then we will need to start expediting the introduction of some of the processes described in https://github.com/AdoptOpenJDK/TSC/issues/220

**Review by sxa — APPROVED** [review]

**Pull request merged**
